coins/bitcoin/src/bitcoin_wallet.rs

```diff
@@ -365,10 +365,10 @@ impl BitcoinWallet {
         let mut max_address = 0;
         let mut path_builder = HDPath::builder();
         path_builder
-            .with_purpose(purpose)
-            .with_coin_type(coin_type)
-            .with_account(account.to_shortform_num())
-            .with_account_hardened(true);
+            .purpose(purpose)
+            .coin_type(coin_type)
+            .account(account.to_shortform_num())
+            .account_hardened(true);
```

Comment:
I think this should be named hardened_account and have another function no_hardened_account. then there is no boolean that will likely always need to be set to true when using the function account_hardened()

think this will be easier to read and know what to set, open to discussing the actual names we settle on